src/sentry/search/events/fields.py

```diff
@@ -1303,6 +1305,24 @@ def is_accessible(self, acl=None):
             transform="apdex(duration, {satisfaction:g})",
             default_result_type="number",
         ),
+        Function(
+            "apdex_new",
+            transform="""
+                apdex(
+                    multiIf(
+                        equals(tupleElement(project_threshold_config, 1), 'lcp'),
+                        if(has(measurements.key, 'lcp'), arrayElement(measurements.value, indexOf(measurements.key, 'lcp')), NULL),
+                        duration
```

Comment:
nit: since we have the replaces at the end lets clean this up a bit too?
```suggestion
                        equals(
                            tupleElement(project_threshold_config, 1),
                            'lcp'
                        ),
                        if(
                             has(measurements.key, 'lcp'),
                             arrayElement(measurements.value, indexOf(measurements.key, 'lcp')),
                             NULL
                        ),
                        duration
```